apps/tenant-management-webapp/src/app/lib/autoComplete.ts

```diff
@@ -198,3 +199,21 @@ const getElementSuggestion = (element) => {
   }
   return suggest;
 };
+const getPositionX = (text: string, search: string): number => {
+  const textArr = text.split('\n');
+  for (let i = textArr.length - 1; i >= 0; i--) {
+    if (textArr[i].indexOf(search) > -1) {
+      return i + 1;
+    }
+  }
+  return 10000;
```

Comment:
Could you make this a constant and give it an appropriate name?  Something that would tell us what 10000 means, like MAX_LINE_LENGTH.

Reply:
set a MAX_LINE_LENGTH